cmd/app/main.go

```diff
@@ -23,6 +27,35 @@ var (
 	Pod                pod.Collector
 )
 
+func getMonitoredNamespaces(clientset *kubernetes.Clientset) ([]string, error) {
```

Comment:
Close. This will get our namespaces on init only. We will want to update our cache of namespaces periodically incase need ones were added.

A good example would be this node watch with a shard informer. 
https://github.com/jmcgrath207/k8s-ephemeral-storage-metrics/blob/master/pkg/node/k8s.go#L106

Wouldn't be against following the same pattern and putting that code under pkg/k8s.go